templates/security/cve/_cve-table.html

```diff
@@ -47,71 +57,66 @@
-                  Deferred
-                {% elif statuses[release.codename].status == "ignored" %}
-                  Ignored
-                {% elif statuses[release.codename].status == "pending" %}
-                  Pending
-                {% elif statuses[release.codename].status == "released" %}
-                  Released
-                {% else %}
-                  &mdash;
-                {% endif %}
-                {% if statuses[release.codename].pocket != "esm-infra" and statuses[release.codename].pocket == "esm-apps" %}
-                  <small class="cve-table-cell esm-info">
-                    {% if statuses[release.codename].pocket == "esm-infra" %}
-                      <a href="/advantage">UA Infra/Apps</a>
-                    {% endif %}
-                    {% if statuses[release.codename].pocket == "esm-apps" %}
-                      <a href="/advantage">UA Apps</a>
-                    {% endif %}
-                  </small>
-                {% endif %}
-              </div>
-              <div class="icon-container__icon icon-cve-info">
-                {% if statuses[release.codename].status == 'pending' or statuses[release.codename].status == "not-affected" %}
-                  <i class="p-icon--information cve-tooltip-icon cve-tooltip-status" data-status="{{ statuses[release.codename].status }}"></i>
-                {% endif %}
-              </div>
-            </td>
+                &mdash;
+              </td>
+            {% endif %}
+          {% endfor %}
+        {% else%}
+          {% for release in releases %}
+            {% set status = package.release_statuses[release.codename] %}
+  
+            {% if status %}
+              <td class="cve-table-cell status-{{ status.slug }}">
+                <div class="cve-color-strip--{{ status.slug.lower() }}"></div>
+                <div class="icon-container__text">
+                  {{ status.name or "&mdash;" }}
+                  {% if status.pocket == "esm-infra" %}
+                    <small class="cve-table-cell esm-info"><a href="/advantage">UA Infra/Apps</a></small>
+                  {% endif %}
+                  {% if status.pocket == "esm-apps" %}
+                    <small class="cve-table-cell esm-info"><a href="/advantage">UA Apps</a></small>
+                  {% endif %}
+                </div>
+                <div class="icon-container__icon icon-cve-info">
+                  {% if status.status == 'pending' or status.slug == "not-affected" %}
+                    <i class="p-icon--information cve-tooltip-icon cve-tooltip-status" data-status="{{ status.slug }}"></i>
+                  {% endif %}
+                </div>
+              </td>
             {% else %}
               <td class="cve-table-cell--muted">
                 <div class="cve-color-strip--dne"></div>
                 &mdash;
               </td>
             {% endif %}
           {% endfor %}
```

Comment:
Are there any HTML differences between this block and the previous block? If not, again, we should build this once off a single `selected_releases` list, which should contain all releases if there's no filter releases specified.